addons/binding/org.openhab.binding.sensebox/src/main/java/org/openhab/binding/sensebox/handler/SenseBoxHandler.java

```diff
@@ -34,36 +29,38 @@
 import org.eclipse.smarthome.core.types.State;
 import org.eclipse.smarthome.core.types.UnDefType;
 import org.openhab.binding.sensebox.config.SenseBoxConfiguration;
-import org.openhab.binding.sensebox.internal.ExpiringCache;
+import org.openhab.binding.sensebox.internal.SenseBoxAPIConnection;
 import org.openhab.binding.sensebox.model.SenseBoxData;
-import org.openhab.binding.sensebox.model.SenseBoxDescriptor;
-import org.openhab.binding.sensebox.model.SenseBoxLoc;
 import org.openhab.binding.sensebox.model.SenseBoxLocation;
 import org.openhab.binding.sensebox.model.SenseBoxSensor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import com.google.gson.Gson;
-
 /**
  * The {@link SenseBoxHandler} is responsible for handling commands, which are
  * sent to one of the channels.
  *
  * @author Hakan Tandogan - Initial contribution
+ * @author Hakan Tandogan - Ignore incorrect data for brightness readings
+ * @author Hakan Tandogan - Changed use of caching utils to ESH ExpiringCacheMap
  */
 public class SenseBoxHandler extends BaseThingHandler {
     private Logger logger = LoggerFactory.getLogger(SenseBoxHandler.class);
 
     protected SenseBoxConfiguration thingConfiguration;
 
-    private Gson gson = new Gson();
-
     private SenseBoxData data = new SenseBoxData();
 
     ScheduledFuture<?> refreshJob;
 
     private final int CACHE_EXPIRY = 10 * 1000; // 10s
 
+    private final ExpiringCacheMap<String, SenseBoxData> CACHE = new ExpiringCacheMap<String, SenseBoxData>(CACHE_EXPIRY);
```

Comment:
Agreed, and will fix, but maybe we should change YahooWeather as well to have a consistent handling of the cache overall:

https://github.com/eclipse/smarthome/pull/3628